Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml

```diff
@@ -163,7 +163,8 @@
         mask:
         - FullTileMask
         layer:
-        - TableLayer
+        - HighImpassable
+        - MidImpassable
   - type: InteractionPopup
     interactSuccessString: fence-rattle-success
```

Comment:
Does LowImpassable work instead?

Reply:
Opaque makes it so both flying and small mobs can pass, but then you can't shoot through it with a laser

Reply:
Considering it's a fence I think it's better to block small mobs than flying.

Reply:
I'm fine with that too, it's either a fence with holes big enough for mice to pass that reaches the ceiling, or a chain fence that doesn't reach the ceiling with extremely tiny "chains". Let's do the latter.